aqua/values.yaml

```diff
@@ -6,4 +6,4 @@ global:
   imagePullPolicy: IfNotPresent
   imagePullSecretName: dockerhub
   dbServicePort: 5432
-  dbExternalServiceHost: "myHost.host.com" #If this value is <String>false the databse will create from the helm chart otherwise specify external host.
+  dbExternalServiceHost: "false" #If this value is <String>false the databse will create from the helm chart otherwise specify external host.
```

Comment:
Would be nicer to use empty string rather than "false" to indicate creating a new database service. 
Comment is not quite English - "If this value is false, a database will be created from the Helm chart, otherwise specify the address of the external database".